index.html

```diff
@@ -0,0 +1,192 @@
+<!DOCTYPE html><html>
+
+  <head> <title>YouTube</title>
+      <link rel="stylesheet" href="style.css" type="text/css">
+  </head>
+  <body>
+      <div class="nav-bar">
+          <div class="logo">
+              <img src="https://pbs.twimg.com/profile_images/1148327441527689217/1QpS06D6.png" id="logo-img" alt="Home">
+          </div>
+          <div class="search-bar">
+              <form class="search-form" action="index.html" method="post">
+                  <input type="text" name="search-bar" value="submit" id="search-text">
+                  <button type="submit">
+                      <img src="https://img.icons8.com/cotton/2x/search--v1.png" alt="" id="search-img">
+                  </button>
+              </form>
+
+          </div>
+          <div class="rightmost-icons">
+              <img src="https://www.freeiconspng.com/uploads/upload-icon-22.png" alt="">
+              <img src="https://icon-library.net/images/icon-apps-download/icon-apps-download-1.jpg" alt="">
+              <img src="https://www.pinclipart.com/picdir/middle/393-3933336_fb-min-324x3242x-messenger-icon-png-white-clipart.png" alt="">
+          </div>
+      </div>
+
+   <div class="video-content">
```

Comment:
[OPTIONAL]
You can use the main tag here instead of the div tag. Using the main tag for the main content of your page it's also a best practice and also make your code more semantic.